scipy/signal/signaltools.py

```diff
@@ -2888,9 +2895,10 @@ def decimate(x, q, n=None, ftype='iir', axis=-1, zero_phase=None):
         The axis along which to decimate.
     zero_phase : bool, optional
         Prevent phase shift by filtering with `filtfilt` instead of `lfilter`
-        when `ftype` is not 'fir'.  A value of `True` is recommended, since a
-        phase shift is generally not desired. Using `None` defaults to `False`
-        for backwards compatibility.
+        when using an IIR filter, and shifting the outputs back by the filter's
+        group delay when using an FIR filter.  A value of `True` is
+        recommended, since a phase shift is generally not desired. Using `None`
+        defaults to `False` for backwards compatibility.
```

Comment:
oh yeah, this default is going to change, right? Better to mention here (in addition to the warning you raise) that the `None` default behavior will change in a subsequent release, so better to set it explicitly.